### PR TITLE
Formating changes for triage_process.md

### PR DESCRIPTION
**Pull request by NickLaMuro**
- Uses reference style markdown links to reduce clutter
- Formats the table to make it easier to view in plain text

### CODE REVIEW

**Comment by NickLaMuro** [comment]
@miq-bot assign @mfeifer 

Sorry if this is overstepping, but I was reading this as it was referenced in some of the miq-bot stuff you assigned me earlier, and this was bugging me...

Happy to close if this is not your thing (as this is more your doc than mine), but figured I would make some quick suggestions in PR form to get some thoughts on it.

cc @Fryguy

File: triage_process.md [hunk 1]
```diff
@@ -39,16 +39,32 @@ Stale pull requests|[Core/Provider stale pull requests](https://github.com/pulls
...
+[UI/API unassigned pull requests]:                          https://github.com/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+-label%3A%22help+wanted%22+repo%3AManageiq%2Fmanageiq-ui-classic+repo%3AManageiq%2Fmanageiq-api+repo%3AManageIQ%2Fui-components+repo%3AManageIQ%2Freact-ui-components+repo%3AManageiq%2Fmanageiq-ui-service+is%3Aopen+no%3Aassignee+
+[UI/API assigned pull requests without scope label]:        https://github.com/issues?page=2&q=is%3Apr+-label%3A%22help+wanted%22+-label%3Arefactoring+-label%3A%22technical+debt%22+-label%3Abug+-label%3Aenhancement+repo%3AManageiq%2Fmanageiq-ui-classic+repo%3AManageiq%2Fmanageiq-api+repo%3AManageIQ%2Fui-components+repo%3AManageIQ%2Freact-ui-components+repo%3AManageiq%2Fmanageiq-ui-service+is%3Aopen&utf8=%E2%9C%93
+[UI/API stale pull requests]:                               https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+label%3Astale+archived%3Afalse+user%3AManageIQ+
```
Comment (NickLaMuro) [inline triage_process.md]: Worth noting, but I was unsure if this was intentional:

This link is exactly the same as `Core/Provider stale pull requests`, so not sure if that was by design (or there is no better way to filter in GH) or if that was a mistake.

**Comment by mfeifer** [comment]
thanks, NIck!

**Comment by NickLaMuro** [comment]
> `@miq-bot` assign `@mfeifer`

Oh, I just realized the bot might not monitor this repo...

**Comment by mfeifer** [comment]
You're right about the last link.  I'm happy to fix that once this is merged.

**Pull request merged**